frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/attachmentSelectTable.jspf

```diff
@@ -24,11 +24,14 @@
 <%--These variables are used as a trick to allow referencing constant values in EL expressions below--%>
 <c:set var="tableModeLicenseInfo" value="<%=PortalConstants.PROJECT_LINK_TABLE_MODE_LICENSE_INFO%>"/>
 <c:set var="tableModeSourceBundle" value="<%=PortalConstants.PROJECT_LINK_TABLE_MODE_SOURCE_BUNDLE%>"/>
+<c:set var="tableModeLicenseInfoForOnlyApprovedAttachment" value="<%=PortalConstants.PROJECT_LINK_TABLE_MODE_LICENSE_INFO%>"/>
```

Comment:
Duplicate variable. same as `tableModeLicenseInfo`

Reply:
Thanks @smrutis1 for the review, removed the duplicate variable and reused the existing variable.